hack/generator/pkg/astmodel/testdata/ConvertBetweenOptionalEnumAndOptionalBaseType.golden

```diff
@@ -0,0 +1,43 @@
+// Code generated by k8s-infra. DO NOT EDIT.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package vCurrent
+
+import "github.com/Azure/k8s-infra/hack/generated/_apis/Verification/vNext"
+
+type Person struct {
+	sku *string `json:"sku"`
+}
+
+// ConvertFromVNext populates our Person from the provided source Person
+func (person Person) ConvertFromVNext(source *vNext.Person) error {
+
+	// sku
+	var sku string
+	if source.sku != nil {
+		sku = string(*source.sku)
+	} else {
+		sku = ""
+	}
+	person.sku = &sku
+
+	// No error
+	return nil
+}
+
+// ConvertToVNext populates the provided destination Person from our Person
+func (person Person) ConvertToVNext(destination *vNext.Person) error {
+
+	// sku
+	var sku string
+	if person.sku != nil {
+		sku = *person.sku
+	} else {
+		sku = ""
+	}
+	skuTemp := Bucket(person.sku)
+	destination.sku = &skuTemp
```

Comment:
Similarly here - `destination.sku` should be nil if `person.sku` is nil.

Reply:
Ditto.